packages/okta-react-native/ios/OktaSdkBridge.swift

```diff
@@ -0,0 +1,373 @@
+                promiseRejecter(OktaReactNativeError.oktaOidcError.errorCode, error.localizedDescription, error)
+                return
+            }
+            
+            guard let currStateManager = stateManager else {
+                let error = OktaReactNativeError.noStateManager
+                promiseRejecter(error.errorCode, error.errorDescription, error)
+                return
+            }
+            
+            currStateManager.writeToSecureStorage()
+            let dic = [
+                OktaSdkConstant.RESOLVE_TYPE_KEY: OktaSdkConstant.AUTHORIZED,
+                OktaSdkConstant.ACCESS_TOKEN_KEY: stateManager?.accessToken
+            ]
+            
+            promiseResolver(dic)
+        }
+    }
+    
+    @objc(signOut:promiseRejecter:)
+    func signOut(_ promiseResolver: @escaping RCTPromiseResolveBlock, promiseRejecter: @escaping RCTPromiseRejectBlock) {
+        
+        let view = RCTPresentedViewController();
+        
+        guard let oidcConfig = config, let currOktaOidc = oktaOidc else {
+            let error = OktaReactNativeError.notConfigured
+            promiseRejecter(error.errorCode, error.errorDescription, error)
+            return
+        }
+        
+        guard let currView = view else {
+            let error = OktaReactNativeError.noView
+            promiseRejecter(error.errorCode, error.errorDescription, error)
+            return
+        }
+        
+        guard let stateManager = OktaOidcStateManager.readFromSecureStorage(for: oidcConfig) else {
+            let error = OktaReactNativeError.unauthenticated
+            promiseRejecter(error.errorCode, error.errorDescription, error)
+            return
+        }
+        
+        currOktaOidc.signOutOfOkta(stateManager, from: currView) { error in
+            if let error = error {
+                promiseRejecter(OktaReactNativeError.oktaOidcError.errorCode, error.localizedDescription, error)
+                return
+            }
+            
+            let dic = [
+                OktaSdkConstant.RESOLVE_TYPE_KEY: OktaSdkConstant.SIGNED_OUT
+            ]
+            stateManager.clear()
+            promiseResolver(dic)
+        }
+    }
+    
+    @objc(getAccessToken:promiseRejecter:)
+    func getAccessToken(promiseResolver: @escaping RCTPromiseResolveBlock, promiseRejecter: @escaping RCTPromiseRejectBlock) {
+        
+        guard let oidcConfig = config else {
+            let error = OktaReactNativeError.notConfigured
+            promiseRejecter(error.errorCode, error.errorDescription, error)
+            return
+        }
+        
+        guard let stateManager = OktaOidcStateManager.readFromSecureStorage(for: oidcConfig) else {
+            let error = OktaReactNativeError.unauthenticated
+            promiseRejecter(error.errorCode, error.errorDescription, error)
+            return
+        }
+        
+        let dic = [
+            OktaSdkConstant.ACCESS_TOKEN_KEY: stateManager.accessToken
```

Comment:
`accessToken ` is optional. Do you need additional guard?

Reply:
additional guard added